agent/main.go

```diff
@@ -40,7 +41,7 @@ func main() {
 		debug bool
 	)
 
-	flag.StringVar(&id, "id", "", "Shim task id")
+	flag.StringVar(&id, "id", "", "Shim task id (required)")
```

Comment:
The thought here is that this id is the ContainerId?

Reply:
yes, i'll change the text

Reply:
This doesn't completely address #35 because the id isn't known currently at the time the agent starts up. That could be changed in the future though so this change still makes sense.

Reply:
interesting.. 

I figured you knew what the containerid was at this point, is this more an anonymous agent in the vm that will act as a proxy for new container(s) at some later time? We generally know the containerid before we create the task&shim to run the container in other flows.

Reply:
At the moment, the agent starts upon VM startup, with no knowledge of the container. However, this will clearly need to change as we move toward a model in which multiple containers (e.g. a pod) shares a VM. So it's not unreasonable to assume that the ID will be known ahead of time by the agent.